src/features/dashboard/Dashboard.tsx

```diff
@@ -1,83 +1,91 @@
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [])
+	useEffect(() => {
+		getAllIntegrations();
+		resetIntegrations();
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
 
-    const cards: ICard[] = [
-        {
-            value: integrations === undefined || integrations.length === 0 ? t('empty') : integrations.length.toString(),
-            content: integrations !== undefined && integrations.length === 1 ? t('oneIntegration') : t('integrations'),
-            links: [
-                {name: t('links.integration'), href: '/integration/new'}
-            ]
-        },
-        {
-            value: totalActive === 0 ? t('empty') : totalActive.toString(),
-            content: totalActive === 1 ? t('oneActiveIntegration') : t('activeIntegrations'),
-            links: [
-                {name: t('links.integrations'), href: '/integration/list'}
-            ]
-        },
-        {
-            value: totalDispatched === 0 ? t('empty') : totalDispatched.toString(),
-            content: totalDispatched === 1 ? t('oneInstance') : t('instances'),
-            links: [
-                {name: t('links.instances'), href: '/integration/instance/list'}
-            ]
-        },
-        {
-            value: totalErrors === 0 ? t('empty') : totalErrors.toString(),
-            content: totalErrors === 1 ? t('oneError') : t('errors'),
-            links: [
-                {name: t('links.instances'), href: '/integration/instance/list'}
-            ]
-        }
-    ]
+	const cards: ICard[] = [
+		{
+			value:
+				integrations === undefined || integrations.length === 0
+					? t("empty")
+					: integrations.length.toString(),
+			content:
+				integrations !== undefined && integrations.length === 1
+					? t("oneIntegration")
+					: t("integrations"),
+			links: [{ name: t("links.integration"), href: "/integration/new" }],
+		},
+		{
+			value: totalActive === 0 ? t("empty") : totalActive.toString(),
+			content:
+				totalActive === 1 ? t("oneActiveIntegration") : t("activeIntegrations"),
+			links: [{ name: t("links.integrations"), href: "/integration/list" }],
+		},
+		{
+			value: totalDispatched === 0 ? t("empty") : totalDispatched.toString(),
+			content: totalDispatched === 1 ? t("oneInstance") : t("instances"),
+			links: [
+				{ name: t("links.instances"), href: "/integration/instance/list" },
+			],
+		},
+		{
+			value: totalErrors === 0 ? t("empty") : totalErrors.toString(),
+			content: totalErrors === 1 ? t("oneError") : t("errors"),
+			links: [
+				{ name: t("links.instances"), href: "/integration/instance/list" },
+			],
+		},
+	];
 
-    return (
-        <PageTemplate id={'dashboard'} keyPrefix={'pages.dashboard'} customHeading>
-            <Box id={'dashboard-card-container'} display="flex" position="relative" width={1} height={1}>
-                {cards.map((card: ICard, index) => {
-                    return (
-                        <DashboardCard
```

Comment:
liker de nye korta, kanskje sette en minWidth så de har samme størrelse? Kanskje også sørge for at de fyller plassen helt til enden, eller kanskje det ser rart ut? 
Altså at første kort starter der boksen under starter, veldig røff sketch her

![image](https://github.com/FINTLabs/fint-flyt-integration-frontend/assets/10653250/afbb1bfa-078b-4c68-a785-fba97486d426)

Reply:
kult at du liker de - minWidth fra MUI ? eventuelt,  Jeg ser at wrapperene fra Aksel og de godtar ikke minWidth eller Width/Heihgt. men ser noe om overridable components - har du kikka på de?

Reply:
tenkte å bruke style propen. 

altså f.eks style={{minWidth: "...px"}}
det godtas i alle akselkomponentene også afaik

Har ikke titta på overridable components

Reply:
![Skjermbilde 2024-01-04 kl  20 14 57](https://github.com/FINTLabs/fint-flyt-integration-frontend/assets/51415032/a10099bc-71ef-4374-8453-64f0303a56d6)

gjør et forsøk med HSTACK

Reply:
Det ser jo råsmud ut 👍